specs/tools/contentProcessing/GltfUtilitiesSpec.ts

```diff
@@ -148,4 +148,154 @@ describe("GltfUtilities", function () {
     expect(data.jsonData.length).not.toBe(0);
     expect(data.binData).toEqual(Buffer.from([]));
   });
+
+  it("replaceLegacyGaussianSplattingExtensionGltf properly upgrades the legacy 3DGS glTF JSON to the current 3DGS glTF spec", function () {
```

Comment:
I opted for a "snapshot" style test here since Jasmine is very good about telling you exactly where some JSON isn't matching correctly.